js/clinicalTimeline.js

```diff
@@ -884,24 +884,32 @@ window.clinicalTimeline = (function(){
               tickValues.push(i * maxTime.daysPerYear);
           }
       } else if (zoomLevel === "months") {
-          tickValues.push(parseInt(beginning));
-          for (i=minTime.m + minTime.y * 12 + 1; i < maxTime.m + maxTime.y * 12 - 1; i++) {
+          if (minTime.d !== 0) {
+            beginning -= minTime.d < 0 ? daysPerMonth : -daysPerMonth;
+            minTime = daysToTimeObject(parseInt(beginning));
+            ending += maxTime.d > 0 ? daysPerMonth : -daysPerMonth;
+            maxTime = daysToTimeObject(parseInt(ending));
+          }
+          for (i=minTime.m + minTime.y * 12; i <= maxTime.m + maxTime.y * 12 ; i++) {
               tickValues.push(i * maxTime.daysPerMonth + parseInt(i/12) * 5);
           }
+          beginning = tickValues[0];
+          ending = tickValues[tickValues.length-1];
+          minTime = daysToTimeObject(parseInt(beginning));
+          maxTime = daysToTimeObject(parseInt(ending));
```

Comment:
You don't need to change beginning/ending/minTime/maxTime anymore here, seems like they are not used further down. Or?